cmd/generic-handlers.go

```diff
@@ -65,10 +65,10 @@ func (h requestSizeLimitHandler) ServeHTTP(w http.ResponseWriter, r *http.Reques
 	h.handler.ServeHTTP(w, r)
 }
 
-// Reserved bucket.
+// Reserved keyword and paths.
 const (
-	minioReservedBucket     = "minio"
-	minioReservedBucketPath = "/" + minioReservedBucket
+	minioReservedKeyword    = "minio"
```

Comment:
why change this? only change that is needed is in `isMinioReservedBucket`  also add a global list 

```
var reservedBucketNames  = []string{"minio", "login"}
```